shoop/core/telemetry.py

```diff
@@ -0,0 +1,196 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from django.conf import settings
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.serializers.json import DjangoJSONEncoder
+
+from django.utils.crypto import get_random_string
+from django.utils.encoding import force_text
+from django.utils.timezone import now
+from shoop.core.models.persistent_cache import PersistentCacheEntry
+import json
+import platform
+import requests
+import shoop
+import sys
+
+OPT_OUT_KWARGS = dict(module="telemetry", key="opt_out")
+INSTALLATION_KEY_KWARGS = dict(module="telemetry", key="installation_key")
+LAST_DATA_KWARGS = dict(module="telemetry", key="last_data")
```

Comment:
Why you're using `dict(foo=bar)` rather than the normal `{'foo': bar}` construct here?